### ISSUE TITLE
Standard library

### ISSUE THREAD

**Issue by cedmundo** [cedmundo/deprecated-bee#6]
Today the standard library only contains the following functions: 
* `pair(a, b)` makes a new pair
* `head(a)` fetches the head of a pair
* `tail(a)` fetches the tail of a pair
* `typename(x)` returns the type name of x as string
* `print(x)` prints to stdout the value of x

Some functions that would be nice to have:
* `open(...)`, `close(...)`, `write(...)`, `read(...)` File control
* `import(...)`, `unimport(...)`, `ldload(...)`, `ldunload(...)` Module control
* `debug(...)` to debug stuff

This issue might depend on #5, I'm not quite sure if I want a standard library as modules or just those functions as minimal and then let the embedding environment to define the rest of the functions.
